class-vip-support-user.php

```diff
@@ -163,6 +168,8 @@ public function __construct() {
 
 		add_filter( 'wp_redirect',          array( $this, 'filter_wp_redirect' ) );
 		add_filter( 'removable_query_args', array( $this, 'filter_removable_query_args' ) );
+		add_filter( 'user_email',           array( $this, 'filter_vip_support_email_aliases' ) );
```

Comment:
In which contexts does this filter fire? I don't see it on the developer site so I'm guessing it's a dynamic filter?

Reply:
Yes dynamic filter, it fires in `sanitize_user_field()` when getting user data for display from a WP_User object https://github.com/WordPress/WordPress/blob/727cdc94c902b72731fd462bab2cf47629a0674a/wp-includes/class-wp-user.php#L320

https://github.com/WordPress/WordPress/blob/c31e8cf756aaec88b27ca175049ecbc859b35011/wp-includes/user.php#L1238